cli/tests/pcluster/test_commands.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2013-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This module provides unit tests for the functions in the pcluster.commands module."""
+
+import pytest
+from botocore.exceptions import ClientError
+
+from assertpy import assert_that
+from pcluster.commands import _create_bucket_with_batch_resources
+from tests.common import MockedBoto3Request
+
+
+@pytest.fixture()
+def boto3_stubber_path():
+    """Specify that boto3_mocker should stub calls to boto3 for the pcluster.utils module."""
+    return "pcluster.utils.boto3"
+
+
+@pytest.mark.parametrize(
+    "region,client_error,log_message",
+    [
+        ("eu-west-1", "An error occurred", "Unable to create internal bucket"),
+        # TODO add a working test
+    ],
+)
+def test_create_bucket_with_batch_resources(region, client_error, log_message, boto3_stubber, caplog):
+    """Verify that create_bucket_with_batch_resources behaves as expected."""
+    bucket_name = "test"
+    resources_dir = "."
+    expected_params = {"Bucket": bucket_name}
+    if region != "us-east-1":
```

Comment:
Could you explain here with a comment the reason of this region check?

Reply:
done